templates/page.html

```diff
@@ -0,0 +1,211 @@
+{% extends 'layout.html' %}
+
+{% block title %}{{ this.title }}{% endblock %}
+
+{% block body %}
+<section class="intro-section">
+  <div class="vertical-center">
+    <div class="container">
+      <div class="row">
+        <div class="headline col">
+          <div><img src="/static/img/logo-reana.svg"></div>
+          <h6>Reproducible research data analysis platform</h6>
```

Comment:
Perhaps a bit smaller intro icon, and spaced away from the "Reproducible..." slogan, so that the motto would stand out more clearly? (Currently people may oversee the slogan since it is too close to the big logo, making the eyes of people going straight into Flexible/Scalable/etc part of screen...)